providers/implementations/signature/dsa.c

```diff
@@ -50,7 +51,7 @@ typedef struct {
     DSA *dsa;
     size_t mdsize;
     /* Should be big enough */
```

Comment:
Probably this comment (and similarly above) is no longer required.

Reply:
Fixed